.gitignore

```diff
@@ -0,0 +1,9 @@
+# Taken from https://github.com/bazel-contrib/rules-template/blob/main/.gitignore
+
+bazel-*
+.bazelrc.user
+.idea/
+.ijwb/
```

Comment:
I wouldn't bother with any of these, but you can keep `bazel-*` if you really want. (I'm not thrilled about `.bazelrc.user`, which explicitly disagrees with `user.bazelrc`, which is the documented Bazel best practice. And, you know, pybind11 has no `.bazelrc` yet anyway.)

Reply:
Ok, I removed all of them